packages/node_modules/@ciscospark/plugin-wdm/src/device/device.js

```diff
@@ -250,6 +250,10 @@ const Device = SparkPlugin.extend({
 
       const timer = setTimeout(() => {
         this.spark.request({
+          headers: {
```

Comment:
I think changes went in that mean you shouldn't need to explicitly omit these headers. try running your tests without this block.